docs/source/conf.py

```diff
@@ -233,6 +233,6 @@
 
 extlinks = {
     'github-demo': (
-        'https://github.com/enthought/envisage/tree/master/envisage/examples/demo/%s',   # noqa: E501
+        'https://github.com/enthought/envisage/tree/main/envisage/examples/demo/%s',   # noqa: E501
```

Comment:
same comment as pyface/traitsui - we shouldn't be linking to examples on the `main` branch, we should be linking to specific versions!